lib/better_errors/error_page.rb

```diff
@@ -112,7 +113,11 @@ def inspect_value(obj)
     end
 
     def inspect_raw_value(obj)
-      value = CGI.escapeHTML(obj.inspect)
+      if defined?(ObjectSpace.memsize_of) 
```

Comment:
This `if` condition should:
1. Make sure `ObjectSpace` exists, in case it's not supported on the version of Ruby being run.
2. See if `ObjectSpace` responds to `memsize_of`. Right now you're calling `ObjectSpace.memsize_of` and then seeing if the return value is defined.